app/controllers/document_lead_image_controller.rb

```diff
@@ -14,15 +14,26 @@ def create
 
     image_uploader = ImageUploader.new(params.require(:image))
 
-    if image_uploader.valid?
-      image = image_uploader.upload(document)
-      redirect_to edit_document_lead_image_path(params[:document_id], image.id)
-    else
+    unless image_uploader.valid?
       redirect_to document_lead_image_path, alert: {
         "alerts" => image_uploader.errors,
         "title" => t("document_lead_image.index.error_summary_title"),
       }
+      return
```

Comment:
Might make your intentions clearer if you did `return redirect_to ...`

Reply:
Hmm...I'd go the other way around, as the use of 'return <value>' to me implies we're actually returning something that's used by something else.